examples/server/spring-server/src/main/kotlin/com/expediagroup/graphql/examples/server/spring/instrumentation/TrackTimesInvokedInstrumentation.kt

```diff
@@ -33,22 +33,22 @@ import java.util.concurrent.ConcurrentHashMap
  * Adds field count tracking in the instrumentation layer if [com.expediagroup.graphql.examples.directives.TrackTimesInvoked] is present.
  */
 @Component
-class TrackTimesInvokedInstrumentation : SimpleInstrumentation() {
+class TrackTimesInvokedInstrumentation : Instrumentation {
```

Comment:
per deprecation notice - should we use `SimplePerformantInstrumentation` instead?

Reply:
the diff is the interface defaults new method to call deprecated methods, from `SimplePerformantInstrumentation` javadoc

> An implementation of Instrumentation that does nothing. It can be used as a base for derived classes where you only implement the methods you want to. The reason this class is designated as more performant is that it does not delegate back to the deprecated methods and allocate a new state object per call.
> 
> This behavior was left in place for backwards compatibility reasons inside Instrumentation and SimpleInstrumentation but has not been done in this class since no existing classes could have derived from it. If you want more performant behavior on methods you don't implement then this is the base class to use, since it will not delegate back to old methods and cause a new state to be allocated.

Reply:
will update dataloader instrumentations as well then.